generate_index.py

```
@@ -0,0 +1,197 @@
+        a:hover {{ text-decoration: underline; color: #2980b9; }}
+        .file-path {{ font-size: 0.8em; color: #95a5a6; display: block; margin-top: 2px; }}
+
+        .footer {{ margin-top: 50px; font-size: 0.8em; color: #7f8c8d; text-align: center; border-top: 1px solid #eee; padding-top: 20px; }}
+    </style>
+</head>
+<body>
+
+    <h1>Documentation Index</h1>
+
+    <div class="tab">
+"""
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)